src/app/personside/visittkort/ShortcutListener.tsx

```diff
@@ -16,22 +22,30 @@ class ShortcutListener extends React.Component<Props> {
     }
 
     componentDidMount() {
-        document.addEventListener('keyup', this.handleShortcut);
+        document.addEventListener('keydown', this.handleShortcut);
```

Comment:
Har det noe å si om det er keyup eller keydown? Dette kommer til å brekke feature togglen på hurtigtasten i gamle modia.

Reply:
Hvorfor vil det brekke?

Leste denne på stackoverflow: https://stackoverflow.com/questions/37557990/detecting-combination-keypresses-control-alt-shift

> Use keydown because Chrome does not fire the keypress event for known keyboard shortcuts.

Kan godt gjøre om hvis det brekker altså.

Reply:
Ah! Kan heller fikse det i backenden, da. Det brekker fordi vi må disable denne shortcuten hvis den nye brukerprofilen er slått av.